### PR TITLE
docs: provide additional explanatory context to README.md

### PR DESCRIPTION
**Pull request by martenpayne**
Expands on the existing README content with some additional context about components/tools used and their purpose.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,38 @@
...
+
+This compbination of project management, build execution, and CDK constructs reduce the effort involved in prototyping new ideas on AWS, and allow developers to seamlessly grow these ideas beyond the prototype phase without requiring a complete re-implementation of the original solution. In addition to the foundation of project configuration and build dependency management provided by the nx-monorepo project, PDK also provides a number of other projen projects (some of which implement common infrastructure patterns using CDK) and stand-alone CDK constructs, which are described below.
```
Comment (JeremyJonas) [inline README.md]: "compbination" => "combination"

File: README.md [hunk 2]
```diff
@@ -1,8 +1,38 @@
 # aws-prototyping-sdk
 
-AWS Prototyping SDK (PDK) is a package which contains useful CDK and Projen constructs.
+## Introduction
```
Comment (JeremyJonas) [inline README.md:3]: Should we add disclaimer that PDK is currently pre-release, and expecting v1 mid-year?

File: README.md [hunk 3]
```diff
@@ -1,8 +1,38 @@
...
+
+## Core Modules
+
+### [nx-monorepo](./packages/nx-monorepo/README.md)
+
+Modern applications are often implented across multiple packages or libraries, each potentially implemented in a different language. PDK provides a [Projen](https://github.com/projen/projen) project called 'nx-monorepo' to simplify and standardise configuration and build across packages in this context. Together with the project-as-code configuration management provided by Projen, this project utilises [Nx Build](https://nx.dev/) to give devlopers the ability to manage a collection of different projects within a single repository. Nx Build allows for explicit dependency tracking between projects, and implements an efficient build caching mechanism to support fast, parallel incremental builds in a language agnostic manner. By combining Projen and NX Build, 'no-monorepo' reduces the effort involved in maintaining and building related projects implemented in a mix of languages.
```
Comment (JeremyJonas) [inline README.md]: Consider rewording - "simplify and standardise configuration and build across packages in this context" to be more readable.

Comment (JeremyJonas) [inline README.md]: "project-as-code configuration management" to simply "project-as-code", which is inherently "configuration management"?

Comment (JeremyJonas) [inline README.md]: "explicit dependency" - and "implicit" for Nx

Comment (JeremyJonas) [inline README.md]: Consider: "Nx Build allows for explicit [and implicit] dependency [management] between polyglot projects, and provides shared caching and parallel task execution for super fast builds across multiple languages with ease."

Comment (JeremyJonas) [inline README.md]: Spelling: 'no-monorepo' => 'nx-monorepo'

File: README.md [hunk 4]
```diff
@@ -1,8 +1,38 @@
 # aws-prototyping-sdk
 
-AWS Prototyping SDK (PDK) is a package which contains useful CDK and Projen constructs.
+## Introduction
 
-## Getting Started
+The AWS Prototyping SDK (PDK) aims to accelerate the development of prototypes on AWS. To achieve this, it provides building blocks for common patterns together with development tools to manage and build your projects. The constructs are based on [AWS CDK](https://github.com/aws/aws-cdk) and implement an expanding number of common application/infrastructure patterns, such as the ability to generate complete client and infrastructure code for a REST API from an OpenAPI specification. The project and build tools the PDK provide allow you manage the configuration of multiple related projects in a language-agnostic way, and to efficiently excute parallel incremental builds.
```
Comment (agdimech) [inline README.md]: execute*

File: README.md [hunk 5]
```diff
@@ -1,8 +1,38 @@
...
+
+This combination of project management, build execution, and CDK constructs reduce the effort involved in prototyping new ideas on AWS, and allow developers to seamlessly grow these ideas beyond the prototype phase without requiring a complete re-implementation of the original solution. In addition to the foundation of project configuration and build dependency management provided by the nx-monorepo project, PDK also provides a number of other projen projects (some of which implement common infrastructure patterns using CDK) and stand-alone CDK constructs, which are described below. Note: PDK is currently pre-release, targetting version 1.0 for mid-2023.
```
Comment (agdimech) [inline README.md:7]: allows*

File: README.md [hunk 6]
```diff
@@ -1,8 +1,38 @@
...
+
+### ['open-api-gateway'](./packages/open-api-gateway/README.md)
+
+This module provides a projen project that allows you to define an API using either Smithy or OpenAPI v3, and a construct which manages deploying this API in API Gateway, given an integration (eg a lambda) for every operation. It generates type-safe CDK constructs, client, and server code to help you rapidly implement and integrate with your API.
+
+### ['pipeline'](./packages/pipeline/README.md)
+
+This module provides a projen project that uses a construct based on CDK's CodePipeline construct, named PDKPipeline, to deploy a CI/CD pipeline. It additionally creates a CodeCommit repository and by default is configured to build the project assumming nx-monorepo is being used (although this can be changed). A Sonarqube Scanner can also be configured to trigger a scan whenever the synth build job completes successfully. This Scanner is non-blocking and as such is not instrumented as part of the pipeline.
+
+### ['identity'](./packages/identity/README.md)
```
Comment (agdimech) [inline README.md:43]: missing pdk-nag and cdk-graph plugins.

**Review by JeremyJonas — APPROVED** [review]

**Review by agdimech — APPROVED** [review]
Thanks Marty!

**Pull request merged**